### ISSUE TITLE
[Adaptive Cards - iOS - Action.Submit.json]: On typing any character from the keypad, form edit fields UI is getting disappeared.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8423]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptiva caeds

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1.Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements control and activate it.
6. Navigate to 'Action.Submit.json' and activate it.
7. Card will appear.
8. Navigate to the 'What is your first name' edit field and double tap on it.
9. Type some character and observe the issue.

**Actual Result:**
Complete UI under 'Action.Submit' is getting hide while typing any character from the keypad.

**Expected Result:**
After typing any character from the keypad, UI should not be hide and inserted text should be displayed properly.

**Note:**
This issue is repro throughout the application.

**User Impact:**
It will impact the screen reader dependent user as they won't be able to know whether text is getting inserted or not?

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/228533527-d7195251-9c45-423e-969b-785005188e3f.MP4

### Card JSON

```JSON
NA
```
